code/datums/diseases/advance/symptoms/asthmosthia.dm

```diff
@@ -0,0 +1,40 @@
+/*
+//////////////////////////////////////
+asthmosthia
+
+	Very very Noticable.
+	Decreases resistance.
+	Decreases stage speed.
+	Reduced transmittability.
+BONUS
+	Spawns gases
+//////////////////////////////////////
+*/
+
+/datum/symptom/asthmothia
+
+	name = "Asthmothia"
+	stealth = -5
+	resistance = -5
```

Comment:
Raise resistance up to like -2 at least and leave stealth at -5 it honestly doesn't matter that much.

Otherwise this will just be another spontaeneous combustion/necrotising f whatever - theoretically scary, cured by salt.